assets/components/subscriptionBundles/paperDigital.jsx

```diff
@@ -3,33 +3,46 @@
 import React from 'react';
 import SubscriptionBundle from 'components/subscriptionBundle/subscriptionBundle';
 import { gridImageProperties } from 'components/threeSubscriptions/helpers/gridImageProperties';
-import { displayPrice, sendTrackingEventsOnClick } from 'helpers/subscriptions';
-import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
 import type { ComponentAbTest } from 'helpers/subscriptions';
+import { discountedDisplayPrice, displayPrice, sendTrackingEventsOnClick } from 'helpers/subscriptions';
+import { flashSaleIsActive } from 'helpers/flashSale';
 
 // ----- Types ----- //
 
 type PropTypes = {|
   url: string,
-  countryGroupId: CountryGroupId,
   abTest: ComponentAbTest | null,
   gridId: 'paperDigitalCirclePink' | 'paperDigitalCircleOrange'
 |};
 
 
 // ----- Component ----- //
 
+function getCopy() {
+  if (flashSaleIsActive('Paper')) {
```

Comment:
Should this be `flashSaleIsActive('PaperAndDigital')` perhaps?

Reply:
Good spot!